sycl/test/esimd/simd_copy_to_copy_from.cpp

```diff
@@ -0,0 +1,78 @@
+// RUN: %clangxx -fsycl -fsyntax-only -Xclang -verify %s
+
+// This test checks that both host and device compilers can:
+// - successfully compile simd::copy_to and simd::copy_from APIs
+// - emit an error if argument of an incompatible type is used
+//   in place of the accessor argument
+
+#include <CL/sycl.hpp>
+#include <CL/sycl/INTEL/esimd.hpp>
+#include <limits>
+#include <utility>
+
+using namespace sycl::INTEL::gpu;
+using namespace cl::sycl;
+
+#ifdef __SYCL_DEVICE_ONLY__
+#define __SYCL_DEVICE_ATTR __attribute__((sycl_device))
+#else
+#define __SYCL_DEVICE_ATTR
+#endif // __SYCL_DEVICE_ONLY__
+
+// --- Postive tests.
+
+void kernel1(accessor<int, 1, access::mode::read_write,
```

Comment:
I think it's better to mark those functions with `SYCL_ESIMD_FUNCTION` even though it's not enforced now.

Reply:
right, thanks.